templates/webapi/test/tr_job_result.html.ep

```diff
@@ -1,4 +1,4 @@
-<td id="res_<%= $resultid %>">
+<td id="res_<%= $resultid %>" name="<%= $jobid %>">
```

Comment:
There's only one row per job and job IDs are unique. So why not using an id, e.g. `id="job-row-<%= $jobid %>"`? Then you could use `document.getElementById('job-row' + array[i])` in the JavaScript code.

Reply:
Sorry, I do not understand your meaning clearly. Do you mean I can use `id` in `<td>` or `<tr>`?  `<td>` already has an id `<td id="res_<%= $resultid %>">`.  And one <tr> includes more than one job, each ARCH has a job.

Reply:
Oh, I see what you mean. Then better stick with the name attribute. Although you could at least use a more unique name (not just the job ID but prefix it with e.g. `job-td-`.